src/frontends/pytorch/src/transforms/aten_getitem_replacer.cpp

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "aten_getitem_replacer.hpp"
+
+#include <memory>
+#include <utility>
+
+#include "openvino/frontend/pytorch/visibility.hpp"
+#include "openvino/op/util/framework_node.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "pt_framework_node.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace pass {
+
+AtenGetItemReplacer::AtenGetItemReplacer() {
+    auto getitem = ov::pass::pattern::wrap_type<ov::op::util::FrameworkNode>();
+
+    ov::matcher_pass_callback callback = [](ov::pass::pattern::Matcher& m) {
+        auto getitem = cast_fw_node(m.get_match_root(), "aten::__getitem__");
+        if (!getitem)
+            return false;
+        auto getitem_index_ptr = getitem->input(1).get_source_output().get_node_shared_ptr();
+        auto getitem_index_const = std::dynamic_pointer_cast<opset8::Constant>(getitem_index_ptr);
+        auto index_val = getitem_index_const->cast_vector<int64_t>();
+
+        auto input_node = getitem->input(0).get_source_output().get_node_shared_ptr();
+        if (auto torch_split = cast_fw_node(input_node, "aten::split")) {
+            if ((torch_split->get_input_source_output(1).get_shape()) == Shape{}) {
```

Comment:
Same comment, check if shape is dynamic

Reply:
Added check:
`
if (torch_split->input(1).get_partial_shape().is_dynamic()) {
                return false;
            }
`

Reply:
Done